multilib/fakepo.py

```diff
@@ -4,13 +4,18 @@ class FakePackageObject(object):
     framework herein
     """
     def __init__(self, po=None, d=None):
-        # FPO's can be created from yum Package Objects or dictionaries
+        # FPO's can be created from yum/dnf Package Objects or dictionaries
         if po:
             self.name = po.name
             self.arch = po.arch
             self.provides = po.provides
             self.requires = po.requires
-            self.files = po.returnFileEntries()
+            try:
+                # yum
+                self.files = po.returnFileEntries()
+            except AttributeError:
+                # dnf
+                self.files = po.files
```

Comment:
Should this be a try/except or does it makes sense to have it as a new named parameter that defaults to yum for backwards compat? That way things using the api can specify which one they want/need based on configuration.